content/desktop/hardened-desktop/_index.md

```diff
@@ -27,18 +27,33 @@ grid:
 >
 >Hardened Docker Desktop is available to Docker Business customers only.
 
+## What is Hardened Docker Desktop?
+
 Hardened Docker Desktop is a group of security features for Docker Desktop, designed to improve the security of developer environments without impacting developer experience or productivity.
 
-It is for security conscious organizations who don’t give their users root or admin access on their machines, and who would like Docker Desktop to be within their organization’s centralized control.
+It lets admins define and enforce robust security settings. It guarantees that developers and the containers they deploy are unable to intentionally or unintentionally circumvent these settings. Additionally, Hardened Docker Desktop enhances container isolation, which helps mitigate potential security threats, and ensures that malicious payloads cannot breach the Docker Desktop Linux VM and the underlying host.
```

Comment:
> Hardened Docker Desktop enhances container isolation

I would re-word this as Enhanced Container Isolation is a feature within the Hardened Docker Desktop model but it's up to the org if they enable it